crates/lib/src/pantry_db.rs

```diff
@@ -103,6 +115,24 @@ pub fn which(cmd: &String, conn: &Connection) -> Result<Vec<String>, rusqlite::E
     Ok(rv)
 }
 
+pub fn projects_for_symbol(
+    symbol: &String,
+    conn: &Connection,
+) -> Result<Vec<String>, rusqlite::Error> {
+    let mut stmt = conn.prepare(
+        "
+        SELECT project FROM provides WHERE program = ?1
+        UNION
+        SELECT project FROM aliases WHERE LOWER(alias) = LOWER(?1);"
```

Comment:
you could _store_ in `LOWER()` on the assumption that calling the function one time is cheaper. but you might discard data you want later.

Reply:
yes this would be better

Reply:
thinking again, I guess we should store the correct representation from the yaml in case we need it that way some other time.